Logger/LogFactory.cs

```diff
@@ -1,10 +1,17 @@
-﻿namespace Logger;
+namespace Logger;
 
 public class LogFactory
 {
+    private string? _filePath;
```

Comment:
```suggestion
    private string? _FilePath;
```
For this class, you should be doing pascal casing for your fields. You will find that this guideline is highly debated in the community.